### PR TITLE
add transfer file feature

### PR DESCRIPTION
**Pull request by milanac030988**

### CODE REVIEW

File: setup_call.bat
Comment (test-fullautomation) [inline setup_call.bat:1]: Hi Cuong,
Thank you for the pull-request.

uups, this file is not supposed to be here.
Please fix.

Thank you,
 Thomas

**Review by test-fullautomation — CHANGES_REQUESTED** [review]
Hi Cuong,
please check my findings.
Thank you,
 Thomas

File: setup_call.bat
Comment (milanac030988) [reply setup_call.bat:1]: Hello Thomas,

I've removed.

Thank you, 
Cuong

Comment (test-fullautomation) [reply setup_call.bat:1]: Hi Cuong,
setup_call.bat is still there?
Please fix.
Thank you,
 Thomas

**Review by test-fullautomation — CHANGES_REQUESTED** [review]

File: setup_call.bat
Comment (milanac030988) [reply setup_call.bat:1]: Hello Thomas,

Sorry that I've committed wrong repo.
It has been deleted now.

Thank you,
Cuong

**Review by test-fullautomation — APPROVED** [review]
Hi Cuong,
looks good to me!
Thank you,
 Thomas

**Pull request merged**